app/src/examples/WorkletRuntimeExample.tsx

```diff
@@ -122,6 +130,72 @@ function PerformanceNowDemo() {
   return <Button title="performance.now" onPress={handlePress} />;
 }
 
+function BackgroundQueueFromJSDemo() {
+  const handlePress = () => {
+    const runtime = createWorkletRuntime('foo');
+    const queue = createBackgroundQueue('bar');
+    runOnBackgroundQueue(queue, runtime, () => {
+      'worklet';
+      console.log('Hello from background!', Math.random());
+    })();
+  };
+
+  return <Button title="Background queue from JS" onPress={handlePress} />;
+}
+
+function BackgroundQueueFromUIDemo() {
+  const handlePress = () => {
+    const runtime = createWorkletRuntime('foo');
+    const queue = createBackgroundQueue('bar');
+    runOnUI(() => {
+      'worklet';
+      const x = Math.random();
+      console.log('Hello from UI thread!', x);
+      runOnBackgroundQueue(queue, runtime, () => {
+        'worklet';
+        console.log('Hello from background!', x);
+      })();
+    })();
+  };
+
+  return <Button title="Background queue from UI" onPress={handlePress} />;
+}
+
+let queue: BackgroundQueue | undefined;
+let runtime: WorkletRuntime | undefined;
+
+function BackgroundQueueLongRunningTaskDemo() {
+  const handlePress = () => {
+    if (queue === undefined) {
+      queue = createBackgroundQueue('bar');
+    }
+    if (runtime === undefined) {
+      runtime = createWorkletRuntime('foo');
+    }
```

Comment:
Is it possible that one would already be destroyed but not the other? Or they don't get destroyed if you get out of the scope?

Reply:
`queue` and `runtime` are global variables defined in lines 164-165 so technically they never get destroyed in this example.